src/main/zapHomeFiles/hud/tools/pageAlertsHigh.js

```diff
@@ -16,6 +16,7 @@ const PageAlertsHigh = (function () {
 	ICONS.PA = 'page-alerts-high.png';
 	const ALERT_TYPE = 'page-alerts';
 	const ALERT_RISK = 'High';
+	const ALERT_RISK_TEXT = I18n.t('alerts_risk_high');
```

Comment:
Maybe "label" instead of "text"?

Reply:
i made this change! thank you for the suggestion.